### PR TITLE
🧪 Add tests for FullInterval

### PR DESCRIPTION
**Pull request by marsop**
🎯 **What:**
Added a missing test suite for `Marsop.Ephemeral.Core.FullInterval<TBoundary, TLength>` to cover constructor behavior, mapping to base properties, `DefaultMeasure`, and `ToString` behavior. 

📊 **Coverage:**
The following scenarios are now covered:
- Constructor correctly initializing start/end bounds and inclusion flags.
- `DefaultMeasure` appropriately delegating length calculation to the overridden `Operator`.
- `ToString` accurately deferring to the underlying base implementation formatting.

✨ **Result:**
The test coverage for `FullInterval` and the `AbstractMeasuredInterval` hierarchy is improved, guaranteeing more robustness when refactoring or tweaking standard interval records.

---
*PR created automatically by Jules for task [5893715293762932804](https://jules.google.com/task/5893715293762932804) started by @marsop*

### CODE REVIEW

File: Marsop.Ephemeral.Tests/Core/Implementation/FullIntervalTests.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        var interval = new TestFullInterval(5, 15, true, true);
+
+        // Act
+        var measure = interval.DefaultMeasure();
+
+        // Assert
+        measure.Should().Be(10);
+    }
+
+    [Fact]
```
Comment (marsop) [inline Marsop.Ephemeral.Tests/Core/Implementation/FullIntervalTests.cs]: @jules Include all combinations of open and closed here, not just one random one

File: Marsop.Ephemeral.Tests/Core/Implementation/FullIntervalTests.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        interval.End.Should().Be(end);
+        interval.StartIncluded.Should().Be(startIncluded);
+        interval.EndIncluded.Should().Be(endIncluded);
+    }
+
+    [Fact]
+    public void DefaultMeasure_UsesOperatorToCalculateMeasure()
+    {
+        // Arrange
+        var interval = new TestFullInterval(5, 15, true, true);
```
Comment (marsop) [inline Marsop.Ephemeral.Tests/Core/Implementation/FullIntervalTests.cs:58]: @jules Also test with negative values and "point" intervals
